src/backend/lima.ts

```diff
@@ -550,6 +550,11 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
       },
     });
 
+    // RD used to store additional keys in lima.yaml that are not supported by lima (and no longer used by RD).
+    // They must be removed because lime intends to switch to strict YAML parsing, so typos can be detected.
```

Comment:
I know you just moved the line, but maybe fix the typo?

```suggestion
    // They must be removed because lima intends to switch to strict YAML parsing, so typos can be detected.
```